pkgs/by-name/be/bevy_cli/package.nix

```diff
@@ -0,0 +1,40 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+
+  pkg-config,
+  openssl,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "bevy_cli";
+  version = "v0.1.0";
+
+  src = fetchFromGitHub {
+    owner = "TheBevyFlock";
+    repo = "bevy_cli";
+    rev = "lint-${version}";
```

Comment:
use tag

Reply:
The tag is called `lint-v0.1.0` 😵‍💫

https://github.com/TheBevyFlock/bevy_cli/releases/tag/lint-v0.1.0

I think this is due to the fact the rust workspace repositories with multiple releasable crates do this automatically if you use `cargo-release` as tooling.